bin/cyhy-import

```diff
@@ -26,13 +26,15 @@ import sys
 # Third-Party Libraries
 import dateutil
 from docopt import docopt
+from collections import defaultdict
 import progressbar as pb
 import netaddr
```

Comment:
I would like this to be updated correspondingly as this is the only thing we're using. Uses of `netaddr.IPSet` would need to be updated as well.

```suggestion
from netaddr import IPSet
```